crates/red_knot_python_semantic/src/types.rs

```diff
@@ -2308,6 +2305,7 @@ impl<'db> Type<'db> {
                 invalid_expressions: smallvec::smallvec![InvalidTypeExpression::BareAnnotated],
                 fallback_type: Type::unknown(),
             }),
+            Type::KnownInstance(KnownInstanceType::ClassVar) => Ok(Type::unknown()),
```

Comment:
This should be a TODO, no?

Reply:
Not exactly sure what you meant by that comment, but I added the following TODO comment:
```rs
// TODO: A bare `ClassVar` should rather be treated as if the symbol was not
// declared at all.
```

Reply:
Sorry, I should have spelled out what (in my mind) the TODO is. I don't think the added comment is quite right, because a bare `ClassVar` is not equivalent to "not annotated". It means "this attribute is a pure ClassVar (that is, not settable on instances) but its declared type must be inferred from the RHS and unioned with Unknown." The latter part of this is equivalent to "not annotated" but the former part is not.